src/kudu/consensus/leader_election.cc

```diff
@@ -239,17 +242,15 @@ void FlexibleVoteCounter::FetchTopologyInfo() {
 }
 
 FlexibleVoteCounter::FlexibleVoteCounter(
-    const std::string& candidate_uuid,
-    int64_t election_term,
-    const LastKnownLeaderPB& last_known_leader,
-    RaftConfigPB config,
+    const std::string &candidate_uuid, int64_t election_term,
+    const LastKnownLeaderPB &last_known_leader, RaftConfigPB config,
```

Comment:
Yes, I ran clang format (with meta configs) on the diff.

Reply:
All changes are in the flexi raft code and I even though it's a pain right now having these changes clang formatted is good in the long term since we can run clang format before every diff to be consistent. Also, I ran the clang format only on the diff changes so it only changes code around the lines that I've touched. However, if these changes are making the review very hard I can try to revert them. Let me know either way.

Reply:
Will do.

Reply:
Nevermind, running clang and rebasing over that is going to be a mess, I'll just remove the clang formatting from this patch manually.